net.rpdev.OpenTodoList.json

```diff
@@ -0,0 +1,60 @@
+{
+    "id": "net.rpdev.OpenTodoList",
+    "rename-icon": "OpenTodoList",
+    "rename-desktop-file": "OpenTodoList.desktop",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.12",
+    "sdk": "org.kde.Sdk",
+    "command": "OpenTodoList",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--socket=pulseaudio",
+        "--socket=session-bus",
```

Comment:
You should run the app by passing `--log-session-bus` and get the list of the Bus's you might need to talk to.